api.md

```diff
@@ -1746,23 +1735,17 @@ GROUP BY <time_bucket( <const_value>, <partition_col_of_hypertable> ),
 #### Sample Usage [](continuous_aggregate-create-examples)
 Create a continuous aggregate view.
 ```sql
-CREATE VIEW continuous_aggregate_view( timec, minl, sumt, sumh )
-WITH ( timescaledb.continuous,
-    timescaledb.refresh_lag = '5 hours',
-    timescaledb.refresh_interval = '1h' )
-AS
+CREATE MATERIALIZED VIEW continuous_aggregate_view( timec, minl, sumt, sumh )
+WITH (timescaledb.continuous) AS
     SELECT time_bucket('1day', timec), min(location), sum(temperature), sum(humidity)
         FROM conditions
         GROUP BY time_bucket('1day', timec)
```

Comment:
I would standardize on 2 space indents.  The section actually has a mix (this is old issue =)

Reply:
I reformatted this code block and the next code block.